packages/shared/src/components/modals/BasePostModal.module.css

```diff
@@ -12,8 +12,6 @@
   }
 
   & :global(.post-content) {
-    @apply bg-theme-bg-primary;
```

Comment:
This is still not needed?
Not sure what it was for tbh

Reply:
The class `post-content` is used on `SharePostModal` and `SquadPostContent`.
But from what I could see, the only property that had any effect, outside of background colour) is the padding-bottom. I could transfer those to tailwind where applicable instead, and then remove this old class.

https://github.com/dailydotdev/apps/blob/eb791b63c112de987c40c5b813c90d75cc70b5fa/packages/shared/src/components/modals/SharePostModal.tsx#L36

https://github.com/dailydotdev/apps/blob/eb791b63c112de987c40c5b813c90d75cc70b5fa/packages/shared/src/components/post/SquadPostContent.tsx#L87

Reply:
Also removed the background from a flew more places I found that it would crash (modal post skeleton loader).